lobby_server.yaml

```diff
@@ -0,0 +1,16 @@
+- version: 1.8.0.9
```

Comment:
note, if we drop the version, then it'll be the 'default' lobby in case the client version does not match any defined in this file. For example, if client version has a future version then we could send them to the test lobby.